oci/private/tarball.sh.tpl

```diff
@@ -7,10 +7,27 @@ readonly BLOBS_DIR="{{blobs_dir}}"
 readonly TAGS_FILE="{{tags}}"
 readonly TARBALL_MANIFEST_PATH="{{manifest_path}}"
 
-REPOTAGS=()
-# read repotags file as array and prepend it to REPOTAGS array.
-IFS=$'\n' REPOTAGSFILE=($(cat "$TAGS_FILE"))
```

Comment:
I think you are overdoing it here.  IFS can be set in a subshell `(  )` which only affects the subshell context where `cat` runs here. 

 you just need 
```
REPOTAGSFILE=($(IFS=$'\n' cat "$TAGS_FILE"))
```

Reply:
@thesayyn Ok I updated the PR to use the subshell. The outer parenthesis was actually breaking the zero-tags case but this worked:
```
REPOTAGS=$(IFS=$'\n' cat "$TAGS_FILE")
```

Reply:
would you like to update the PR accordingly? I'd like to get you on our contributiors list